src/functions.php

```diff
@@ -28,3 +28,16 @@ function format_date_for_json(\DateTimeImmutable $date)
 
      return $result.'Z';
  }
+
+ function space_cache_key($id)
+ {
+     return \sprintf('space-%s.json', $id);
+ }
+
+ /**
+  * @param $id
+  */
+ function content_type_cache_key($id)
+ {
+     return \sprintf('contentType-%s.json', $id);
+ }
```

Comment:
I don't think that the cache keys should have the `.json` extension, because it's something specific to a cache implementation.